opentok-whiteboard.js

```diff
@@ -341,6 +386,12 @@ var OpenTokWhiteboard = angular.module('opentok-whiteboard', ['opentok'])
                         if (drawHistory.length > 0 && event.connection.connectionId !==
                                 OTSession.session.connection.connectionId) {
                             batchSignal('otWhiteboard_history', drawHistory, event.connection);
+                            // Iterate through redo stack to hide any paths which have been undone
+                            if (redoStack.length) {
+                                redoStack.forEach(function(id) {
+                                    sendUpdate('otWhiteboard_undo', id);
```

Comment:
Yeah, this won't work without unique ids for each of the undo operations. Right now it will just undo the most recent operations. 

Also, you probably don't want to broadcast these updates to everyone in the session which is what sendUpdate will do. You probably want to just send it to `event.connection` like the history.